code/modules/shuttle/special.dm

```diff
@@ -310,6 +310,14 @@
 	name = "fight pit bear"
 	desc = "This bear's trained through ancient Russian secrets to fear the walls of its glass prison."
 	environment_smash = ENVIRONMENT_SMASH_NONE
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/bear/fightpit/Initialize()
+	. = ..()
+	var/M = round(GLOB.player_list.len / 20, 0.1)
```

Comment:
you should use descriptive var names

Reply:
also should use length(SSticker.mode.current_players[CURRENT_LIVING_PLAYERS]) and 20 should be a define probably